lib/api/api.js

```diff
@@ -6,6 +6,7 @@ import bucketDelete from './bucketDelete';
 import bucketGet from './bucketGet';
 import bucketGetACL from './bucketGetACL';
 import bucketGetVersioning from './bucketGetVersioning';
+import bucketGetWebsite from './bucketGetWebsite';
```

Comment:
I added these tests to the Integration PR for the remaining bucket website actions: https://github.com/scality/Integration/pull/365